Source/Server/Managers/ChatManager.cs

```diff
@@ -267,10 +285,14 @@ public static string GetUsernameFromMention(string mention)
             return mention.Replace("@", "");
         }
 
-        public static void ShowChatInConsole(string username, string message)
+        public static void ShowChatInConsole(string username, string message, bool fromDiscord = false)
         {
             if (!Master.serverConfig.DisplayChatInConsole) return;
```

Comment:
Extract's common logic. (Message handling/logging) Seperates the methods to help reduce redundancy and improve readability/simplify.

public static void ShowChatInConsole(string username, string message, bool fromDiscord = false)
{
    if (!Master.serverConfig.DisplayChatInConsole) return;
    Logger.Message($"{(fromDiscord ? "[Discord]" : "[Chat]")} > {username} > {message}");
}